python/ray/tests/test_pydantic_serialization.py

```diff
@@ -146,17 +150,14 @@ def hello(v: str, u: User) -> str:
     ray.get(ray.put(make))
 
 
-def test_serialize_app_imported_closure(start_ray):
-    from pydantic_module import closure
-
+@pytest.mark.parametrize("BaseModel", [BaseModelV1, BaseModelV2])
+def test_serialize_app_imported_closure(start_ray, BaseModel: Type):
     ray.get(ray.put(closure))
 
 
-@pytest.mark.skipif(
-    not IS_PYDANTIC_2,
-    reason="Test fails with Pydantic 1.10.12, but succeeds with Pydantic 1.9.2.",
-)
-def test_serialize_serve_dataclass(start_ray):
+# TODO: why only new one.
```

Comment:
Could you clarify this `TODO`?

Reply:
still WIP i'll add a comment

Reply:
I'll leave this as a `TODO` for now since it's not a regression.